src/pages/index.md

```diff
@@ -5,125 +5,89 @@ description: This is the overview page of Cat Analytics
-
-Below you can find links to the Adobe Pass Authentication Flows documentation and API References. For more detailed information, please refer to the documentation linked in the resources section.
-
-## Discover
-
-### Adobe Pass Flows
-
-#### Basic Flows
-
-##### [Primary Device Profiles](https://experienceleague.adobe.com/en/docs/pass/authentication/programmer-integration-guide/rest-api-v2/rest-api-v2-flows/rest-api-v2-basic-flows/rest-api-v2-basic-profiles-primary-application-flow.html)
-
-Obtain information on user active logins from the primary streaming application.
-
-##### [Second Screen Profiles](https://experienceleague.adobe.com/en/docs/pass/authentication/programmer-integration-guide/rest-api-v2/rest-api-v2-flows/rest-api-v2-basic-flows/rest-api-v2-basic-profiles-secondary-application-flow.html)
-
-Obtain information on user active logins from a secondary device.
-
-##### [Primary Device Authentication](https://experienceleague.adobe.com/en/docs/pass/authentication/programmer-integration-guide/rest-api-v2/rest-api-v2-flows/rest-api-v2-basic-flows/rest-api-v2-basic-authentication-primary-application-flow.html)
-
-The scenario in which the primary streaming application has the ability to provide all the required parameters in one single call in order to create an authentication session and has the capability to open a user agent in order to authenticate with the selected MVPD.
-
-##### [Second Screen Authentication](https://experienceleague.adobe.com/en/docs/pass/authentication/programmer-integration-guide/rest-api-v2/rest-api-v2-flows/rest-api-v2-basic-flows/rest-api-v2-basic-authentication-secondary-application-flow.html)
-
-The scenario in which the primary streaming application has the capability to create an authentication session, but does not have the ability to gather all the required parameters or open a user agent in order to authenticate with the selected MVPD.
-
-##### [Authorization](https://experienceleague.adobe.com/en/docs/pass/authentication/programmer-integration-guide/rest-api-v2/rest-api-v2-flows/rest-api-v2-basic-flows/rest-api-v2-basic-authorization-primary-application-flow.html)
-
-Verify that a user is entitled to view content represented as resource from the primary streaming application.
-
-##### [Preauthorization](https://experienceleague.adobe.com/en/docs/pass/authentication/programmer-integration-guide/rest-api-v2/rest-api-v2-flows/rest-api-v2-basic-flows/rest-api-v2-basic-preauthorization-primary-application-flow.html)
-
-Verify that a user may have access to a list of resources in order for the primary streaming application to present accurate information to the user on content he would be able to see.
+* [Adobe Pass Authentication Documentation](https://experienceleague.adobe.com/en/docs/pass/authentication/home)
```

Comment:
remove 'Adobe' [Pass Authentication Documentation](https://experienceleague.adobe.com/en/docs/pass/authentication/home)
Comment

Reply:
If we keep the actual length, it's not going to be rendered right